### ISSUE TITLE
Android - Kotlin: Datatypes in Kotlin

### ISSUE THREAD

**Issue by LavleenAgrawal** [girlscript/winter-of-contributing#443]
I want to make documentation / audio content on this, May I ?
Lavleen Agrawal (Batch 5 - Android Kotlin)

**Comment by rahulsain** [comment]
First, use correct format while creating Issue Name.

**Comment by sitamadex11** [comment]
@LavleenAgrawal Any update regarding your work?

**Comment by ayush-sleeping** [comment]
hey @rahulsain @sitamadex11 , its been 13 days this issue opened . Kindly assign me , I want to make Proper Documentation on " 02.1 Datatypes in Kotlin " . (I have completed my 4 issue, Now I have 0 issue assign to me)

**Comment by LavleenAgrawal** [comment]
Yes @sitamadex11 sir my task was completed and pull request issued...changes are assigned and completed it but need some guidance...

**Comment by ayush-sleeping** [comment]
hey, @rahulsain because in past my mic. is not working properly , That's why I removed my self from "assign" . <br>
But now , I have my new mic. and its working properly . <b> Please "Assign" me this again for video on this issue . </b>

**Comment by ayush-sleeping** [comment]
Hey @rahulsain completed video [ #7546 ] part on this issue . 
#

Please assign me this issue for Audio part .

**Comment by rahulsain** [comment]
> Hey @rahulsain completed video [ #7546 ] part on this issue .
> 
> Please assign me this issue for Audio part .

One contributor can contribute to only one part of the same issue

**Comment by ayush-sleeping** [comment]
Please assign me this issue for Audio part . @rahulsain

**Comment by ayush-sleeping** [comment]
> > Hey @rahulsain completed video [ #7546 ] part on this issue .
> > Please assign me this issue for Audio part .
> 
> One contributor can contribute to only one part of the same issue

ok Rahul ,
